app/src/main/java/ch/hikemate/app/model/route/HikesViewModel.kt

```diff
@@ -40,6 +46,20 @@ class HikesViewModel(
     private val dispatcher: CoroutineDispatcher = Dispatchers.IO
 ) : ViewModel() {
   companion object {
+    val Factory: ViewModelProvider.Factory =
+        object : ViewModelProvider.Factory {
+          @Suppress("UNCHECKED_CAST")
+          override fun <T : ViewModel> create(modelClass: Class<T>): T {
+            return HikesViewModel(
+                savedHikesRepo =
+                    SavedHikesRepositoryFirestore(
+                        FirebaseFirestore.getInstance(), FirebaseAuth.getInstance()),
+                osmHikesRepo = HikeRoutesRepositoryOverpass(OkHttpClient()),
+                elevationService = ElevationServiceRepository(OkHttpClient()))
```

Comment:
Nitpick: Couldn't you use the same OkHttpClient?

Reply:
That's a good idea, done 👌🏻

Reply:
Thanks!